Packs/Phishing/ReleaseNotes/3_5_8.md

```diff
@@ -0,0 +1,15 @@
+
+#### Layouts
+
+##### Phishing Incident v3
+
+- Added "Selected indicators to block" section and incident field. this section will show the indicators chosen to be blocked during the investigation.
+
+#### Playbooks
+
+##### Phishing - Generic v3
+
+- "Block Indicators - Generic v2" sub-playbook replaced with "Block Indicators - generic v3" and the conditional task "Should indicators be blocked automatically?" has been removed. Block indicators method will be determined in the sub-playbook itself using the "AutoBlockIndicators" input.
```

Comment:
- "Block Indicators - Generic v2" sub-playbook replaced with "Block Indicators - generic v3" sub-playbook.
- the conditional task "Should indicators be blocked automatically?" has been removed and handled by the new sybplaybook (handled by the `BlockIndicators` input).